templates/user/username.html.twig

```diff
@@ -0,0 +1 @@
+<span>{{ account.name.value }}</span>
```

Comment:
I don't think we want to surface anything from the user object on this site. For news there is a field_author plain text field to be used on the byline.

Reply:
Deleted.